### PR TITLE
Fix/@typescript-eslint/consistent-type-assertions

### PR DESCRIPTION
**Pull request by dw-0**
depends on #3981

### CODE REVIEW

**Comment by dtslvr** [comment]
Can you rebase please? 🙂

**Comment by dw-0** [comment]
*sigh* Something happened to some of the imports in some files. They got reformatted from absolute to relative imports. I need to fix that ...

**Comment by dtslvr** [comment]
> _sigh_ Something happened to some of the imports in some files. They got reformatted from absolute to relative imports. I need to fix that ...

Looks like you have found a way to automate the imports from absolute to relative paths? 😃 Good to know, as this shows a looot of warnings! But probably wait for the other open PRs as this seems to be a bigger intervention.

**Comment by dw-0** [comment]
> > _sigh_ Something happened to some of the imports in some files. They got reformatted from absolute to relative imports. I need to fix that ...
> 
> Looks like you have found a way to automate the imports from absolute to relative paths? 😃 Good to know, as this shows a looot of warnings! But probably wait for the other open PRs as this seems to be a bigger intervention.

Yeah by accident. Seems like i had "optimize imports" enabled in my IDE while editing some of the files. No idea why. I think that was the reason.

Although relative imports are recommended by eslint, NX usually recommends absolute imports. Thats a big topic we have to decide on which way to go in the future, correct :D

File: CHANGELOG.md
Comment (dtslvr) [inline CHANGELOG.md:1]: ❤️

File: apps/client/src/app/components/admin-market-data/admin-market-data.component.ts [hunk 1]
```diff
@@ -71,36 +71,35 @@ export class AdminMarketDataComponent
...
-        type: <Filter['type']>'PRESET_ID'
+        type: 'PRESET_ID' as Filter['type']
       }
     ]);
   public benchmarks: Partial<SymbolProfile>[];
   public currentDataSource: DataSource;
   public currentSymbol: string;
-  public dataSource: MatTableDataSource<AdminMarketDataItem> =
-    new MatTableDataSource();
+  public dataSource = new MatTableDataSource<AdminMarketDataItem>();
```
Comment (dtslvr) [inline apps/client/src/app/components/admin-market-data/admin-market-data.component.ts:102]: 👍🏻

**Review by dtslvr — APPROVED** [review]
Thanks for the PR, @dw-0! 👍🏻

**Comment by dtslvr** [comment]
> Yeah by accident. Seems like i had "optimize imports" enabled in my IDE while editing some of the files. No idea why. I think that was the reason.

Then maybe keep it for the moment, but deactivate the warning rule?

**Comment by dw-0** [comment]
> Then maybe keep it for the moment, but deactivate the warning rule?

Afaik my IDE (IntelliJ IDEA) doesn't care about the warnings itself. It would optimize the imports on its own, even if there are no warnings from ESLint. It's an IDE feature i think. No worries, its fine to have it disabled for now :)

**Comment by dtslvr** [comment]
In _VS Code_ the many yellow warnings from the imports are quite annoying 😏

**Comment by dw-0** [comment]
Absolutely 😄 i think we are on a good way to get rid of them soon

**Pull request merged**